diagnostic.md

```diff
@@ -35,7 +35,7 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    In the top route definition, product is nested under products in the actual file tree. You would need to use products.product when accessing the information elsewhere in the Ember app. In the bottom route, product is NOT nested under products in the file structure. Generally speaking, an Ember url does not necessarily reflect file structure.
```

Comment:
Nested route definitions, do require a nested file structure.  This means that the Route and Template files for `'product'` are expected to be found at `app/products/product`.

One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The not-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.  

These two route definitions are identical  URL paths.